data.js

```diff
@@ -17,6 +17,24 @@ module.exports = [
       'SEO',
     ],
   },
+{
+name: 'Abdulmalik Adekunle',
+    description: 'Front End Web developer, Graphics Designer, IT consultant.',
+    url: 'https://abdulmalik.hashnode.dev',
+    twitter: '@Emeritus_Ade',
+    tags: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+      'Web Development',
+      'React',
+      'Next',
+      'Front End',
+      'NodeJS',
```

Comment:
Hi @Adekunle27,  I'm just checking your tags and I saw that NodeJS is available as Node.js, same for Next that is Next.js.
Could you please change that two ones for:
```
'Node.js',
'Next.js',
```
and also the spacing in the open key `{` and `name` property 🤗
Thank you in advance and thanks for adding your blog!!